src/schema/users.ts

```diff
@@ -1111,6 +1114,16 @@ export const typeDefs = /* GraphQL */ `
       image: Upload!
     ): User! @auth @rateLimit(limit: 5, duration: 60)
 
+    """
+    Upload user resume
+    """
+    uploadResume(
+      """
+      Asset to upload
+      """
+      resume: Upload!
+    ): EmptyResponse @auth @rateLimit(limit: 5, duration: 60)
```

Comment:
5 in a minute sounds a bit much?

Reply:
We set file upload size limit in `/src/index.ts`

```
app.register(MercuriusGQLUpload, {
    maxFileSize: 1024 * 1024 * 20,
    maxFiles: 1,
  });
```

Reply:
Awesome 🚀